src/xtopcom/xwrouter/src/small_net_cache.cc

```diff
@@ -327,7 +207,7 @@ uint32_t SmallNetNodes::AddNode(NetNode node) {
     if (ifind == net_nodes_cache_map_.end()) {
         auto net_nodes = std::make_shared<NetNodes>();
         net_nodes->latest_version = node.m_version;
-        AddNodeLimit(service_type, net_nodes->nodes, node);
+        net_nodes->nodes.push_back(node);
```

Comment:
size of net_nodes->nodes won't greater than the limit?

Reply:
This `net_nodes`(`= std::make_shared<NetNodes>()`) is created if `net_nodes_cache_map_` can't find this kind of service_type, in which the `nodes` is empty.